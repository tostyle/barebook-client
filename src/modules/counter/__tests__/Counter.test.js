import React from 'react'
import ReactDOM from 'react-dom'
import { shallow, mount } from 'enzyme'
import sinon from 'sinon'
import renderer from 'react-test-renderer'
import Counter from '../Counter'

it('example test renders without crashing with enzyme', () => {
  shallow(<Counter />)
})

it('example test renders toMatchSnapshot with Jest', () => {
  const tree = renderer.create(<Counter />)
  expect(tree.toJSON()).toMatchSnapshot()
})

it('example test renders toMatchSnapshot with Enzyme', () => {
  const tree = mount(<Counter />)
  expect(tree).toMatchSnapshot()
})
