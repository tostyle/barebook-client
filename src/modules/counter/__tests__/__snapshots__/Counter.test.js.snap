// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example test renders toMatchSnapshot with Enzyme 1`] = `
<pure(Counter)>
  <Counter
    counter={0}
    decrement={[Function]}
    increment={[Function]}
    incrementIfOdd={[Function]}
  >
    <section>
      <styled.p>
        <p
          className="sc-bdVaJa dQSkNS"
        >
          To get started, edit 
          <code>
            src/routes/index.js 
          </code>
          and save to reload.
        </p>
      </styled.p>
      <p>
        Clicked: 
        0
         times 
        <button
          onClick={[Function]}
        >
          +
        </button>
         
        <button
          onClick={[Function]}
        >
          -
        </button>
         
        <button
          onClick={[Function]}
        >
          Increment if odd
        </button>
      </p>
    </section>
  </Counter>
</pure(Counter)>
`;

exports[`example test renders toMatchSnapshot with Jest 1`] = `
<section>
  <p
    className="sc-bdVaJa dQSkNS"
  >
    To get started, edit 
    <code>
      src/routes/index.js 
    </code>
    and save to reload.
  </p>
  <p>
    Clicked: 
    0
     times 
    <button
      onClick={[Function]}
    >
      +
    </button>
     
    <button
      onClick={[Function]}
    >
      -
    </button>
     
    <button
      onClick={[Function]}
    >
      Increment if odd
    </button>
  </p>
</section>
`;
